net-dns/knot-resolver/files/knot-resolver-6.0.12-pytest_tomllib.patch

```diff
@@ -1,19 +1,11 @@
-From 70d2254de37555167cf60b60d1539ecc9d798fea Mon Sep 17 00:00:00 2001
-From: Nicolas PARLANT <nicolas.parlant@parhuet.fr>
-Date: Mon, 25 Nov 2024 22:21:07 +0000
-Subject: [PATCH] use tomli instead toml for pytest
-
----
- tests/manager/test_knot_resolver_manager.py | 6 +++---
- 1 file changed, 3 insertions(+), 3 deletions(-)
-
+use tomllib instead toml for pytest
```

Comment:
No upstream links or references (even to a gentoo bug)?

Reply:
upstream uses toml and allows python3.8 for now.
added gentoo bug

Reply:
Please say that in the commit message and ideally the patch.

Reply:
Done. I was reluctant to submit upstream a package change with a version condition for a small version test on pyproject.toml

Reply:
That's fine, just it should be clear then in the patch why it's not there. Upstream supporting Python 3.8 is a reasonable excuse to not bother IMO, and I don't think I'd bother in your position. Just tell me that ;)